farcy/handlers.py

```diff
@@ -109,6 +109,10 @@ def assert_usable(self):
         try:
             version = (check_output([self.BINARY, '--version'], stderr=STDOUT)
                        .decode('utf-8'))
+
+            # Remove all non 'versioning' characters from the string
+            version = re.search('(\d+\.*)+', version).group(0)
```

Comment:
This will fail if the output is something like "MyCoolProgram Copyright 2015 is on version 3.5.5"
I am guessing that's rare and we should think about it when/if that problem shows up?